utils/jump_host.py

```diff
@@ -24,23 +71,66 @@ def get_identity_from_vault(self, jh):
         return identity
 
     def init_ssh_base_cmd(self):
+        user_host = '{}@{}'.format(self.user, self.hostname)
+
+        self.set_ssh_base_cmd([
+            'ssh',
+            '-o', 'StrictHostKeyChecking=yes',
+            '-o', 'UserKnownHostsFile={}'.format(self.known_hosts_file),
+            '-i', self.identity_file, '-p', str(self.port), user_host])
+
+    def init_ssh_session(self):
+        session = jumpssh.SSHSession(
+            self.hostname,
+            self.user,
+            private_key_file=self.identity_file,
+            port=self.port,
+        )
+        self.set_ssh_session(session)
```

Comment:
I wouldn't use a setter here, it make it more complicated. init is implicitely a setter...

Reply:
fixed